data/ecosystems/z/zksync.toml

```diff
@@ -25,6 +25,7 @@ sub_ecosystems = [
   "SyncSwap",
   "TrueFi Engineering (Ethworks)",
   "Uniswap",
+  "Vest Exchange",
```

Comment:
You need to create a new ecosystem for "Vest Exchange". For that, you need to create a new toml file like vest-exchange.toml with title "Vest Exchange" and add the org in it (https://github.com/VestLabs). This is assuming VestLabs dedicates most of it efforts to work on a projects(s) in ZKSync. Otherwise, just place here the specific repos that contribute to ZKSync